src/app/frontend/overview/workloadstatus/component.ts

```diff
@@ -12,96 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Component} from '@angular/core';
+import {Component, Input} from '@angular/core';
+import {ResourcesRatio} from '@api/frontendapi';
 
-@Component({
-  selector: 'kd-workload-statuses',
-  templateUrl: './template.html',
-})
+export const emptyResourcesRatio: ResourcesRatio = {
+  cronJobRatio: [],
+  daemonSetRatio: [],
+  deploymentRatio: [],
+  jobRatio: [],
+  podRatio: [],
+  replicaSetRatio: [],
+  replicationControllerRatio: [],
+  statefulSetRatio: []
+};
+
+@Component(
+    {selector: 'kd-workload-statuses', templateUrl: './template.html', styleUrls: ['./style.scss']})
 export class WorkloadStatusComponent {
-  // @Input() ratio: any;
+  @Input() resourcesRatio: ResourcesRatio;
   colors: string[] = ['#00c752', '#f00', '#ffad20', '#006028'];
```

Comment:
I see on the preview that colors are very bright and with hight contrast. Can you use same colors as before?

Reply:
The hex codes are the same as before, I believe the problem is with `AllocationChartComponent`. In v1.10.1, it changes color on hover, which is not the behavior in the current implementation of the component. It needs some more fixes too as @jeefy pointed out in his review. I'm thinking to address this in another PR.

Reply:
@ajatprabha I think we should take the time to get it right on the first try. This isn't time critical, and this is fairly narrow-scoped. :) 

I could be convinced otherwise, but I'd say let's keep it in a single PR.

Reply:
Updated and pushed the changes.